htdocs/s/my.js

```diff
@@ -218,6 +218,15 @@ var evalOrg = {};
 
 	this.livePreviewInit = function(imported){
 		const { PhpWeb } = imported;
+
+		// Compute version
+		const phpVersionRunner = new PhpWeb()
+		phpVersionRunner.addEventListener('output', (event) => {
+			$('#phpversion').textContent = "Output for php " + event.detail[0]
+		});
+		phpVersionRunner.run('<?php echo phpversion();')
```

Comment:
For now we are forced to run php to get this information.

In the future we will be able to use `PhpWeb.phpVersion`

```suggestion
                // Set PHP version in the UI
                $('#phpversion').textContent = "Output for php " + PhpWeb.phpVersion
		phpVersionRunner.run('<?php echo phpversion();')
```

We can "opt-in" to this by using https://cdn.jsdelivr.net/npm/php-wasm@0.0.9-alpha-9/PhpWeb.mjs instead of the current latest version `v0.0.8`.